tuning/utils/utils.py

```diff
@@ -46,3 +46,31 @@ def load_yaml_or_json(file_path: str) -> dict:
         if ext == ".json":
             return json.load(f)
     return None
+
+
+def validate_datasets(datasets):
```

Comment:
I know it is already tested inside the code path but could we have a testcase for this...a simple one should suffice. Thanks

Reply:
Based on [this discussion here](https://ibm-research.slack.com/archives/C07U5DX68EP/p1734624445786499), function `validate_mergeable_datasets` now just logs warnings and allow concatenation of any datasets.